lib/rubocop/cop/metrics/class_length.rb

```diff
@@ -39,8 +39,13 @@ def on_class(node)
         end
 
         def on_casgn(node)
-          _scope, _name, block_node = *node
-          check_code_length(node) if block_node.class_definition?
+          if node.parent&.assignment?
```

Comment:
Oh, right!
It's an interesting pattern, I wonder if it is common?
Maybe it should be abstracted (later) in rubocop-ast, so that for any "assignment" we can get the assignment block with one method call, wdyt?

Reply:
Good point. I'm not sure yet, but I think we can investigate a similar case and consider abstracting it to RuboCop AST (later).